lib/bloc/events/login_event.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:meta/meta.dart';
+import 'package:equatable/equatable.dart';
+
+@immutable
+abstract class LoginEvent extends Equatable {
+  LoginEvent([List props = const []]) : super();
+}
+
+class EmailChanged extends LoginEvent {
+  final String email;
+
+  EmailChanged({@required this.email}) : super([email]);
+
+  @override
+  String toString() => 'EmailChanged { email :$email }';
+
+  @override
+  List<Object> get props => [email];
+}
+
+class PasswordChanged extends LoginEvent {
+  final String password;
+
+  PasswordChanged({@required this.password}) : super([password]);
+
+  @override
+  String toString() => 'PasswordChanged { password: $password }';
+
+  @override
+  List<Object> get props => [password];
+}
+
+class Submitted extends LoginEvent {
+  final String email;
+  final String password;
+
+  Submitted({
+    @required this.email,
+    @required this.password,
+  }) : super([email, password]);
+
+  @override
+  String toString() {
+    return 'Submitted { email: $email, password: $password }';
+  }
+
+  @override
+  List<Object> get props => [email, password];
+}
+
+class LoginWithGooglePressed extends LoginEvent {
```

Comment:
LoginWithCredentialsPressed -> LoginWithCredentials

Reply:
That is the event so maybe we add "pressed" will be cleaner 👍

Reply:
this is unnecessary for cleaning. this event for many cases calling to server. the action pressed just for the UI action.